index.js

```diff
@@ -77,7 +77,10 @@ module.exports = {
 
   testemMiddleware: function(app) {
     if (!this._isCoverageEnabled()) { return; }
-    attachMiddleware(app, { root: this.project.root, ui: this.ui });
+    attachMiddleware(app, {
+      root: this.project.root,
+      configDirName: path.dirname(this.project.configPath())
```

Comment:
On second thought, I think it would be better to simply pass the project down into the middleware and have our config util function accept the project and avoid the path munging throughout the rest of the system.